experimental/packages/opentelemetry-exporter-prometheus/package.json

```diff
@@ -1,6 +1,7 @@
 {
-  "name": "@opentelemetry/exporter-prometheus",
+  "name": "@opentelemetry/exporter-prometheus-wip",
```

Comment:
After the OTLP exporters are updated we should remove wip from all names and have a release

Reply:
No need to change the name here I think. The package is private and nothing depends on it.

Reply:
Reverted :)